vendor.conf

```diff
@@ -1,17 +1,17 @@
 # the following lines are in sorted order, FYI
-github.com/Azure/go-ansiterm 388960b655244e76e24c75f48631564eaefade62
-github.com/Microsoft/hcsshim v0.5.25
+github.com/Azure/go-ansiterm 19f72df4d05d31cbe1c56bfc8045c96babff6c7e
+github.com/Microsoft/hcsshim v0.6.1
 github.com/Microsoft/go-winio v0.4.2
-github.com/Sirupsen/logrus v0.11.0
-github.com/moby/buildkit fed5c1d9cee6f734f58f3addca6e8d1750df48a6
+github.com/moby/buildkit da2b9dc7dab99e824b2b1067ad7d0523e32dd2d9 https://github.com/dmcgowan/buildkit.git
 github.com/davecgh/go-spew 346938d642f2ec3594ed81d874461961cd0faa76
 github.com/docker/libtrust 9cbd2a1374f46905c68a4eb3694a130610adc62a
 github.com/go-check/check 4ed411733c5785b40214c70bce814c3a3a689609 https://github.com/cpuguy83/check.git
 github.com/gorilla/context v1.1
 github.com/gorilla/mux v1.1
-github.com/jhowardmsft/opengcs v0.0.9
+github.com/jhowardmsft/opengcs b9d0120d36f26e981a50bf18bac1bb3f0c2b8fef https://github.com/dmcgowan/opengcs.git
```

Comment:
Merged but cannot use master yet, see conversation with @jhowardmsft about a follow up to bring it to master. I am using a fork here to avoid pulling in unrelated changes of which I know little about. My expectation is those that better understand these projects will be able to get off my forks by opening a follow up updating to the most recent.